src/helpers/mod.rs

```diff
@@ -19,6 +27,25 @@ use tinyvec::ArrayVec;
 pub const MESSAGE_PAYLOAD_SIZE_BYTES: usize = 8;
 type MessagePayload = ArrayVec<[u8; MESSAGE_PAYLOAD_SIZE_BYTES]>;
 
+/// Represents an opaque identifier of the helper instance. Compare with a [`Role`], which
+/// represents a helper's role within an MPC protocol, which may be different per protocol.
+/// `HelperIdentity` will be established at startup and then never change. Components that want to
+/// resolve this identifier into something (Uri, encryption keys, etc) must consult configuration
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct HelperIdentity {
+    id: u8,
```

Comment:
as we discussed, helper identity needs to be an opaque identifier. Maybe u8 is not the right choice for it, once we shape out helper config, we may decide to change it. but something lightweight is needed (this struct must be cloned often)